eval.go

```diff
@@ -237,14 +237,24 @@ nextFileMatch:
 		}
 
 		for ; nextDoc < docCount; nextDoc++ {
+			repoID := d.repos[nextDoc]
+			repoMetadata := &d.repoMetaData[repoID]
+
 			// Skip tombstoned docs
```

Comment:
minor can you update this docstring to say we are skipping repos that are tombstoned